exercises/atbash-cipher/example.rs

```diff
@@ -0,0 +1,39 @@
+fn ascii(ch: char) -> u8 {
+    ch as u8
+}
+
+fn get_transpose(ch: char) -> char {
+    if ch.is_digit(10) {
+        ch
+    } else {
+        (ascii('z') - ascii(ch) + ascii('a')) as char
+    }
+}
+
+pub fn encode(plaintext: &str) -> String {
+    plaintext
+    .to_lowercase()
+    .chars()
+    .filter(|&ch| ch.is_alphanumeric())
+    .filter(|&ch| (ch as u8) < 128)
```

Comment:
`is_ascii` may simplify this: https://doc.rust-lang.org/std/primitive.str.html#method.is_ascii